src/main/java/com/kodilla/ecommercee/controller/UserController.java

```diff
@@ -1,27 +0,0 @@
-package com.kodilla.ecommercee.controller;
-
-import com.kodilla.ecommercee.domain.GenericEntity;
-import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.PutMapping;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
-
-@RestController
-@RequestMapping("/v1/user")
-public class UserController {
```

Comment:
Dlaczego to usuwasz? Nie powinieneś tego robić

Reply:
No i nie wiem, coś mi się porąbało. Ten branch jest za bardzo skopany usunę go i wstawię na nowo.